### ISSUE TITLE
<USER STORY> Users who have screen readers

### ISSUE THREAD

**Issue by Libbu** [Libbu/CyberPixels#4]
As a **user who relies on a screen reader** I want **the images and icons on the website to have meaningful alt text** so that I can **use the site.** 

### Acceptance Criteria

- Acceptance Criteria 1: Images have meaningful alt text. 
- Acceptance Criteria 2: Alt text is available if images are unable to load. 
- Acceptance Criteria 3: Alt text available for icons. 

### Tasks

- [ ] Write meaningful alt text for images and icons on website. 
- [ ] Write meaningful alt text for icons.
